recipes/gumbo-parser/all/conanfile.py

```diff
@@ -0,0 +1,67 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class GumboParserConan(ConanFile):
+    name = "gumbo-parser"
+    description = "An HTML5 parsing library in pure C99"
+    topics = ("conan", "parser")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/google/gumbo-parser"
+    license = "Apache-2.0"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+        if self.settings.compiler == "Visual Studio":
+            raise ConanInvalidConfiguration("%s is not supported by Visual Studio" % self.name)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_folder = "gumbo-parser-{0}".format(self.version)
+        os.rename(extracted_folder, self._source_subfolder)
+
+    def _configure_autotools(self):
+        if not self._autotools:
+            with tools.chdir(self._source_subfolder):
+                self.run("./autogen.sh", win_bash=tools.os_info.is_windows)
```

Comment:
I think you need to add `automake/1.16.2` as a build requirement

Reply:
Done